docs/text.md

```diff
@@ -240,7 +240,7 @@ On Android, these roles have similar functionality on TalkBack as adding Accessi
 
 Tells the screen reader to treat the currently focused on element as being in a specific state.
 
-You can provide one state, no state, or multiple states. The states must be passed in through an object. Ex: `{selected: true, disabled: true}`.
+You can provide one state, no state, or multiple states. The states must be passed in through an object, e.g. `{selected: true, disabled: true}`.
```

Comment:
The abbreviation "Ex:" isn't used anywhere else in the documentation, while "e.g." is used almost 900 times.